QA.md

```diff
@@ -0,0 +1,210 @@
+|---------------------------|--------------|--------------------|
+| Code review               |              | Moderate/High      |
+| Static analysis           | Low          | Low                |
+| Unit testing              | Moderate     | Low                |
+| Fuzz testing              | Low/Moderate | Low                |
+| Model-based testing (MBT) | High         | Low                |
+| End-to-end (E2E) testing  | High         | Low                |
+| Testnet testing           | High         | Moderate/High      |
+| Protocol audits           |              | High               |
+
+## Concerns
+
+Tendermint QA is split across the following concerns:
+- [Correct software engineering](#correct-software-engineering)
+- Protocol correctness
+  - [Consensus](#consensus)
+  - [Mempool](#mempool)
+  - [Evidence](#evidence)
+  - [Peer exchange (PEX)](#peer-exchange-pex)
+  - [Block sync](#block-sync)
+  - [State sync](#state-sync)
+  - [Light client](#light-client)
+- [Application integration (ABCI)](#application-integration-abci)
+- [Network stability](#network-stability)
+- Operational dynamics
+  - [Crash/recovery](#crash-recovery)
+  - [Upgrading](#upgrading)
+
+### Correct software engineering
+
+| Concern                         | Code Review | Static Analysis | Unit Testing | Fuzz Testing | E2E Testing | Testnet Testing |
+|---------------------------------|-------------|-----------------|--------------|--------------|-------------|-----------------|
+| Security                        |             |                 |              |              |             |                 |
+| Memory leaks                    |             |                 |              |              |             |                 |
+| Storage leaks                   |             |                 |              |              |             |                 |
+| Error handling                  |             |                 |              |              |             |                 |
+| Serialization / deserialization |             |                 |              |              |             |                 |
+
+### Protocol correctness
+
+#### Consensus
+
+| Concern                          | Code Review | Static Analysis | Unit Testing | Fuzz Testing | MBT | E2E Testing | Testnet Testing | Protocol Audit |
+|----------------------------------|-------------|-----------------|--------------|--------------|-----|-------------|-----------------|----------------|
+| Code/specification alignment     |             |                 |              |              |     |             |                 |                |
+| - Grammar                        |             |                 |              |              |     |             |                 |                |
+| - Parameters of API functions    |             |                 |              |              |     |             |                 |                |
```

Comment:
These two are quite specific for ABCI++. I would replace them in all tables by "Implementation ensures safety and liveness as given in the specification". Or perhaps something more catchy.